src/components/GrantCard.tsx

```diff
@@ -0,0 +1,202 @@
+import React, { useState} from 'react';
+import { StyleSheet, ViewComponent } from 'react-native';
+import { View, Pressable, Text, Alert, Modal, TouchableHighlight, Image } from 'react-native';
+import { FlipInEasyX } from 'react-native-reanimated';
+import globalStyles from '../globalStyles';
+import ViewContainer from './ViewContainer';
+
+
+
+export const GrantCard = (props : any) => {
+  const [modalVisible, setModalVisible] = useState(false);
+
+  return (
+    <ViewContainer>
+       <Modal
+        animationType="slide"
+        transparent={true}
+        visible={modalVisible}
+        onRequestClose={() => {
+          Alert.alert("Modal has been closed.");
+          setModalVisible(!modalVisible);
+        }}
+      >
+        <View style={styles.centeredView}>
+          <View style={styles.modalView}>
+            <View style={styles.innerModal}>
+              <View>
+                <Text style={styles.modalTitle}>{props.title}</Text>
+              </View>
+              <View>
+                <Text style={styles.modalBodyText}>{props.amount}</Text>
+              </View>
+              <View>
+                <Text style={styles.modalBodyText}>Due Date: {props.deadline}</Text>
+              </View>
+              <View>
+                <Text style={styles.modalBodyText}>Elligible Countries: {props.countries}</Text>
+              </View>
+              <View>
+                <Text style={styles.modalBodyText}>
+                  {props.description}
+                </Text>
+              </View>
+              <View style={styles.images}>
+                <Image
+                  source={require('../../assets/grantDummyImg.png')}
+                />
+              </View>
+              
+            </View>
+            
+            <Pressable
+              style={[styles.button, styles.buttonClose]}
+              onPress={() => setModalVisible(!modalVisible)}
+            >
+              <Text style={styles.closeButtonText}>View Application</Text>
+            </Pressable>
+          </View>
+        </View>
+      </Modal>
+
+      <TouchableHighlight underlayColor={'#942000'} onPress={() => setModalVisible(true)} style={styles.cardContainer}>
+        <View>
+          <Text style={styles.titleText}>{props.title}</Text>
+          <Text style={styles.whiteText}> {props.amount}</Text>
+          <Text style={styles.whiteText}> Due Date: {props.deadline} </Text>    
+        </View>
+    </TouchableHighlight>
+    <Image
+      source={require('../../assets/expand.svg')}/>
```

Comment:
Seems like up to this point, this project has been using SvgXml to handle icons (see `assets/Icons.tsx`). Instead of importing the SVG like this, you should probably stick to that convention